### PR TITLE
Preliminary shakespear txt

### PR DESCRIPTION
**Pull request by jackattackyang**
# Pull Request

## Description

Small preliminary shakespear text, ~ 1MB, from TF tutorial link. Speaker names have been filtered out as well as newline spacing.

## Ticket(s)

Affects #11 

## Todo

[Complete works is around 10MB](http://www.gutenberg.org/files/100/100-0.txt) but format is much messier

### CODE REVIEW

File: scripts/clean_shakespear.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import requests
```
Comment (rwblickhan) [inline scripts/clean_shakespear.py]: I'll be putting up a PR that introduces [pipenv](https://pipenv.readthedocs.io/en/latest/) a bit later tonight; you'll have to add `requests` to the Pipfile that that uses once that happens.

File: data/clean_shakespear/clean_shakespear.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...of rebellion, insolence, sedition, Which we ourselves have plough'd for, sow'd, and scatter'd, By mingling them with us, the honour'd number, Who lack not virtue, no, nor power, but that Which they have given to beggars. Well, no more. No more words, we beseech you. How! no more! As for my country I have shed my blood, Not fearing outward force, so shall my lungs Coin words till their decay against those measles, Which we disdain should tatter us, yet sought The very way to catch them. You speak o' the people, As if you were a god to punish, not A man of their infirmity. 'Twere well We let the people know't. What, what? his choler? Choler! Were I as patient as the midnight sleep, By Jove, 'twould be my mind! It is a mind That shall remain a poison where it is, Not poison any further. Shall remain! Hear you this Triton of the minnows? mark you His absolute 'shall'? 'Twas from the canon. 'Shall'! O good but most unwise patricians! why, You grave but reckless senators, have you thus Given Hydra here to choose an officer, That with his peremptory 'shall,' being but The horn and noise o' the monster's, wants not spirit To say he'll turn your current in a ditch, And make your channel his
```
Comment (rwblickhan) [inline data/clean_shakespear/clean_shakespear.txt]: Is there a value in keeping the full cleaned .txt file in the repo if we have the .zip?

File: scripts/clean_shakespear.py (hunk 1 above)
Comment (jackattackyang) [reply scripts/clean_shakespear.py]: Oh right, ended up directly going with `request`. Forgot to let you know.

File: data/clean_shakespear/clean_shakespear.txt (hunk 2 above)
Comment (jackattackyang) [reply data/clean_shakespear/clean_shakespear.txt]: True, I'll just leave the zip

File: scripts/clean_shakespear.py (hunk 1 above)
Comment (rwblickhan) [reply scripts/clean_shakespear.py]: Yep, no problem! If you want to go ahead and merge this I can actually just make that change later on.

**Comment by jackattackyang** [comment]
> can we fix the spelling to "shakespeare"?

I honestly had no idea haha, thanks

**Comment by rwblickhan** [comment]
The .zip file still has the spelling mistake 😛

**Comment by jackattackyang** [comment]
you guys are too fast

**Pull request merged**